j/jswtf.js

```diff
@@ -0,0 +1,8 @@
+function wtf(a,b,c){
```

Comment:
Is this an actual programing language or is it just an implementation in javascript?